frontend/src/components/applications/review/OperatingCostsSummary.vue

```diff
@@ -51,7 +51,7 @@
       </div>
 
       <!-- UPLOAD DOCUMENTS -->
-      <div v-if="showUploadedDocs" class="mt-4">
+      <div v-if="showUploadedDocs || !isRenewal" class="mt-4">
```

Comment:
It doesn't make sense to have a computed method (which is only used for this dev) and also add explicit conditions to the v-if statement. All the logic should be in the computed method.